spec/unit/graph/simple_graph_spec.rb

```diff
@@ -605,7 +609,7 @@ def graph_to_yaml(graph, which_format)
             edge.keys.each { |x| expect(['source', 'target', 'callback', 'event']).to include(x) }
             %w{source target callback event}.collect { |x| edge[x] }
           end
-          expect(Set.new(actual_edge_tuples)).to eq(Set.new(expected_edge_tuples))
+          expect(Set.new(actual_edge_tuples)).to eq(Set.new(expected_edge_tuples.map { |tuple| tuple.map {|e| e.nil? ? nil : e.to_s }}))
```

Comment:
Why do we need to special case the case where `e` is nil now, but we didn't need to previously? Does `Catalog.from_data_hash` now include `nil` edges (that would be strange)?